core/modules/wdb/QueryRunner.cc

```diff
@@ -135,13 +134,12 @@ bool QueryRunner::runQuery() {
     Release release(_task, this);
 
     if (_task->getCancelled()) {
-        LOGS(_log, LOG_LVL_DEBUG, "runQuery, task was cancelled before it started. "
-             << _task->dbName);
+        LOGF(_log, LOG_LVL_DEBUG, "runQuery, task was cancelled before it started. %1%" % _task->hash);
```

Comment:
Don't do this, we are removing LOGF macros from log and replacing them with LOGS. Same for the message below.